### PR TITLE
Add `exclude|e` argument to handle exlcuding files

### LINKED ISSUE #46: Add ability to exclude files from check
What should be done:
- [ ] Add the command line argument: `-e FILE_NAME`, `--exclude FILE_NAME`;
- [ ] Support multiple use: `-e FILE_1 -e FILE_2 -e FILE_3`;
- [ ] Add usage information to README.

### PR DESCRIPTION
**Pull request by mstruebing**
closes #46

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -82,5 +82,22 @@ fn dotenv_files(matches: clap::ArgMatches) -> Result<Vec<PathBuf>, Error> {
...
+            if let Ok(file) = fs::canonicalize(file) {
+                if paths.contains(&file) {
+                    for (index, path) in paths.clone().iter().enumerate() {
+                        if path == &file {
+                            paths.remove(index);
+                        }
+                    }
+                }
+            }
+        }
```
Comment (mgrachev) [inline src/lib.rs]: You can do it with less code 😉
```rust
let exclude_paths: Vec<PathBuf> = excludes.filter_map(|f| fs::canonicalize(f).ok()).collect();

paths.retain(|p| !exclude_paths.contains(p));
```

**Comment by mgrachev** [comment]
@mstruebing Thanks a lot for your contributing!

I've left a few comments. Could you please fix them?

**Comment by mgrachev** [comment]
I've just updated the file `CHANGELOG.md`. Please pull new changes from the branch `master`.

File: src/lib.rs (hunk 1 above)
Comment (mstruebing) [reply src/lib.rs]: Cool trick, I've searched for a simpler solution but didn't came up with that. :)

**Comment by mgrachev** [comment]
@mstruebing Thanks a lot ❤️

**Pull request merged**